libs/common/src/lib/runtime-track.ts

```diff
@@ -2,6 +2,7 @@ import { getDistance } from 'geolib';
 
 import * as protos from '../protos/track';
 import { diffDecodeArray, diffEncodeArray } from './math';
+import { Score } from "../../../../apps/fxc-front/src/app/logic/score/scorer";
```

Comment:
- cross package imports must use absolute paths, i.e. `@flyxc/...` (see the root `tsconfig.base.json` where there are defined)
- but the common libs should not import from other app packages
- `common` has code that depends on the browser
- `common-node` has code that does not depend on the browser.

One more things for libs: exports should be in `libs/<name>/src/index.ts` - for historical reasons you will see quite a few `export * from './path/to/file';` but named export should be preferred for new code.

If `Score` is needed by any of the lib, it should be moved there.

Reply:
solved. Dependency not necessary